Cargo.toml

```diff
@@ -7,9 +7,12 @@ edition = "2018"
 
 [dependencies]
 async-trait = "0.1"
-fluent-bundle = "0.15"
-fluent-fallback = "0.5"
-fluent-testing = { git = "https://github.com/projectfluent/fluent-rs", optional = true, features = ["sync", "async"] }
+#fluent-bundle = { git = "https://github.com/nordzilla/fluent-rs.git", rev = "94c06c10b511dd968926bb51f72656e7a071fbf1" }
+#fluent-fallback = { git = "https://github.com/nordzilla/fluent-rs.git", rev = "94c06c10b511dd968926bb51f72656e7a071fbf1" }
+#fluent-testing = { git = "https://github.com/nordzilla/fluent-rs.git", rev = "94c06c10b511dd968926bb51f72656e7a071fbf1", optional = true, features = ["sync", "async"] }
+fluent-bundle = { path = "../fluent-rs/fluent-bundle" }
+fluent-fallback = { path = "../fluent-rs/fluent-fallback" }
+fluent-testing = { path = "../fluent-rs/fluent-testing", optional = true, features = ["sync", "async"] }
```

Comment:
Will need to change these once the `fluent-rs` patch lands and is published to `Crates.io`.